dali/ft/filecopy.cpp

```diff
@@ -1326,22 +1326,23 @@ void FileSprayer::checkFormats()
         {
         case FFTfixed:
             if ((tgtType != FFTvariable)&&(tgtType != FFTvariablebigendian))
-                throwError(DFTERR_BadSrcTgtCombination);
+                throwError2(DFTERR_BadSrcTgtCombination, FileFormatTypeStr[srcType], FileFormatTypeStr[tgtType]);
```

Comment:
not sure if possible, but could check srcType and tgtType are in range before switch statement, to prevent crash here if not. i.e. check that they are less than < FileFormatType::FFTlast